src/SonarScanner.MSBuild.Common/ConsoleLogger.cs

```diff
@@ -22,6 +22,9 @@
 using System.Collections.Generic;
```

Comment:
This kind of cleanup in the same PR with the changes makes the review harder than it should be. I think that in the future we should split the cleanup from the actual implementation.

Reply:
Yeah I realized that when it was too late, sorry for that, I'll try to separate the PRs next time